ReactQt/tests/test-slider-props.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2017-present, Status Research and Development GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include "reactpropertytestcase.h"
+#include "reactview.h"
+#include <QDebug>
+#include <QTest>
+#include <QtQuick/QQuickView>
+
+class TestSliderProps : public ReactPropertyTestCase {
+    Q_OBJECT
+
+private slots:
+
+    virtual void initTestCase() override;
+
+protected:
+    virtual QQuickItem* control() const override;
+    virtual QVariantMap propValues() const override;
+};
+
+QQuickItem* TestSliderProps::control() const {
+    return singleControl();
+}
+
+void TestSliderProps::initTestCase() {
+    ReactPropertyTestCase::initTestCase();
+    loadQML(QUrl("qrc:/TestSliderProps.qml"));
+    waitAndVerifyJsAppStarted();
+}
+
+QVariantMap TestSliderProps::propValues() const {
+    return {{"p_value", 0.5},
```

Comment:
Probably it would be nice to cover here all the properties we have in Slider component qml.

Reply:
@vkjr initially I've added to Slider.qml the whole list of props from Slider.js, and finally supposed to extend props unit test only with fully supported props. Others than currently listed props are not really supported, I hope to implement their support in next commits, so will extend unit test also.

Reply:
@MaxRis, ok I see. I usually consider this "props" tests just as a proof that property value passes well between JS and QML, since they don't check any functionality.

Reply:
@vkjr , got it, thanks, Have updated to cover all QML properties